packs/ocm-klusterlet-v0.13.2/charts/ocm-klusterlet/templates/operator.yaml

```diff
@@ -0,0 +1,90 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: klusterlet
+  namespace: open-cluster-management
+  labels:
+    app: klusterlet
+spec:
+  replicas: {{ .Values.replicas }}
+  selector:
+    matchLabels:
+      app: klusterlet
+  template:
+    metadata:
+      annotations:
+        target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
+      labels:
+        app: klusterlet
+    spec:
+      affinity:
+        podAntiAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+          - weight: 70
+            podAffinityTerm:
+              topologyKey: failure-domain.beta.kubernetes.io/zone
+              labelSelector:
+                matchExpressions:
+                - key: app
+                  operator: In
+                  values:
+                  - klusterlet
+          - weight: 30
+            podAffinityTerm:
+              topologyKey: kubernetes.io/hostname
+              labelSelector:
+                matchExpressions:
+                - key: app
+                  operator: In
+                  values:
+                  - klusterlet
+      serviceAccountName: klusterlet
+      securityContext:
+        runAsNonRoot: true
+      containers:
+      - name: klusterlet
+        image: quay.io/open-cluster-management/registration-operator:v0.13.2
+        args:
+          - "/registration-operator"
```

Comment:
These args are fine for the initial draft. 

But its good to think if there will be cases for users to customize additional flags. Generally, if the container accepts more flags than what we've have here, its better to expose an extraArgs and use it here in the template to allow user added flags.